includes/sanitizers/class-amp-playbuzz-sanitizer.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+require_once( AMP__DIR__ . '/includes/sanitizers/class-amp-base-sanitizer.php' );
+
+/**
+ * Converts Playbuzz embed to <amp-playbuzz>
+ */
+class AMP_Playbuzz_Sanitizer extends AMP_Base_Sanitizer{
+
+    public static $tag = 'div';
+    public static $pb_class = 'pb_feed';
+
+
+
+    private static $script_slug = 'amp-playbuzz';
+    private static $height = '500';
+    private static $script_src = 'https://cdn.ampproject.org/v0/amp-playbuzz-0.1.js';
+
+
+    public function get_scripts() {
+        if ( ! $this->did_convert_elements ) {
+            return array();
+        }
+        return array( self::$script_slug => self::$script_src );
+    }
+
+
+    public function sanitize() {
+
+
+        $nodes = $this->dom->getElementsByTagName( self::$tag );
+        $num_nodes = $nodes->length;
+
+
+        if ( 0 === $num_nodes ) {
+
+            return;
+
+        }
+
+
+        for ( $i = $num_nodes - 1; $i >= 0; $i-- ) {
+            $node = $nodes->item( $i );
+
+            if(self::$pb_class !== $node -> getAttribute('class')){
+                continue;
+            }
+
+            $old_attributes = AMP_DOM_Utils::get_node_attributes_as_assoc_array( $node );
+
+            $new_attributes = $this->filter_attributes( $old_attributes );
+
+
+            if(!isset($new_attributes['data-item']) && !isset($new_attributes['src'])){
```

Comment:
Can we please following spacing standards here and through the rest of the file? https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage